android/app/src/main/java/labs/pooh/eaterslab/ui/activity/hello/HelloSelectActivity.kt

```diff
@@ -27,7 +28,7 @@ class HelloSelectActivity : AbstractThemedActivity() {
 
         fabSearch.setOnClickListener {
             markButtonsClickable(false)
-            start<MainActivity> {
+            start<SearchActivity> {
```

Comment:
Unintended change?

Reply:
Making the clean with the app activities  specifications